charts/backbeat/values.yaml

```diff
@@ -47,3 +47,12 @@ replication:
     nodeSelector: {}
     tolerations: []
     affinity: {}
+
+  status:
+    replicaCount: 1
+    
+    resources: {}
+    nodeSelector: {}
+    tolerations: []
+    affinity: {}
+    
```

Comment:
minor: missing newline at end of file

Reply:
Added the newline to the file. Thank you!